### PR TITLE
Extract shortlogs from initial ver to latest ver

### PR DESCRIPTION
**Pull request by reddheeraj**
output of v2.6.0 (only 100 lines)

"Lightweight Patch Manager" (1):
      new list macros for USB

Aaron Baranoff (1):
      Add pci id to tulip net driver

Abraham van der Merwe (1):
      1083/1: 64-bit unsigned modulo arithmetic support I've added support for 64-bit modulo arithmetic on ARM. This is needed for the video4linux API to function properly and since there's already support for 64-bit divides, I think there shouldn't be any reason for the absence of this.

Achim Leubner (1):
      gdth driver update

Adam Belay (77):
      PnP Rewrite V0.9 - 2.5.43
      PnP Rewrite Fixes - 2.5.44
      PnP cleanups and resource changes - 2.5.44 (1/4)
      PnPBIOS changes - 2.5.44 (2/4)
      Convert CS4236B driver - 2.5.44 (3/4)
      update PnP layer to driver model changes - 2.5.44 (4/4)
      Linux PnP Support V0.93 - 2.5.50
      PnP bugfix
      PnP gameport driver update
      Preparations and Cleanups
      This patch contains an improved resource management algorithm.  It is capable of resolving nearly any conflict between two or more PnP devices. It also contains better error reporting and a manual override capability.
      Moves the resource parsing functions to a new location "support.c".  These resource parsing functions contain many improvements including the ability to set resources according to actual value rather than dependent functions. The interface changes will be able to take advantage of this.
      Interface Updates
      ISAPnP Updates
      PnPBIOS Updates
      Trivial Card Service Fix
      Radio-Cadet PnP Update
      IDE PnP Update
      Trivial C99 Update
      PnP Bug Fixes
      OSS Sound Blaster Update from Paul Laufer
      Large Stack Usage Fix
      Resource Management Performance Fix
      Manual Resource Setting Update
      Interface Changes
      Merge http://linux.bkbits.net/linux-2.5 into neo.rr.com:/home/ambx1/src/bk/linus-2.5
      PnP Card Serivice Revisions
      ALS100 Updates
      OSS SB driver Updates
      Aditional Card Service Changes
      PnP Card Service Cleanups
      PnPBIOS Update
      ALS100 Memory Leak Fix
      PnP changes to allow MODULE_DEVICE_TABLE()
      Merge http://linux.bkbits.net/linux-2.5 into neo.rr.com:/home/ambx1/src/bk/linus-2.5
      ALSA SB16 PnP Update
      Silently Ignore if the device is already active/disabled
      Increment Version Number
      Merge http://linux.bkbits.net/linux-2.5 into neo.rr.com:/home/ambx1/linux/bk/linus-2.5
      AZT2320 PnP Updates
      ALSA DT019X PnP Updates
      ALS100: kmalloc params fix
      ES968: kmalloc params fix
      [PNP] Resource Management Cleanups and Updates
      [PNP] /drivers/pnp/resource.c check_region warning fix
      [PNP] Module Compilation Fix
      [PNP] PnPBIOS resource setting fix
      [PNP] re-add the previously removed "get" command in interface.c.
      [PNP] Trivial Typo fix regarding DMAs
      [PNP] Remove some leftover resource config options in isapnp
      [PNP] Important Resource Parsing Fixes
      [PNP] pnp_init_resource_table compile fix
      [PNP] Locking Fixes
      [PNP] Handle Disabled Resources Properly
      [PNP] Allow resource auto config to assign disabled resources
      [PNP] Fix manual resource setting API
      [PNP] Handle unset resources properly
      [PNP] Remove protocol_data from pnp_dev and pnp_card
      [PNPBIOS] Move Parsing Functions to the PnPBIOS driver
      [SOUND] Remove __(dev)initdata from all pnp sound drivers
      [PNPBIOS] Move low level code into a separate file
      [PNP] Remove device naming based on id
      [PNP] Increment Version Number
      [PCMCIA] Fix PnP Probing in i82365.c
      Remove remaining usage of device.name in PnP
      Fix sb_card.c for "name" removal
      Fix awe PnP probing
      [PNPBIOS] compilation fix for pnpbios without proc support
      [PNP] release card devices on probe failure
      [PNPBIOS] move detection code into core.c
      [PNP] remove DMA 0 restrictions
      janitor: remove unneeded includes (isapnp)
      [ISAPNP] remove unused isapnp_allow_dma0 modparam
      [PNPBIOS] return proper error codes on init failure
      [PNPBIOS] move some more functions to local include file
      Fix ISAPNP netdev initialization
      reserve resources specified by the PnPBIOS properly

Adam Bernau (1):
      LSM to designated initializers

Adam J. Richter (10):
      Patch: linux-2.5.7-pre1/net/ipv4/ipmr.c did not compile
      fix do_open() interaction with rd.c
      Building list of drives in right order
      The following patch shaves a six bytes from the loaded size of pcspkr.o and another 90 elsewhere in the .o file.
      linux-2.5.41/drivers/usb/core/hub.c called down() from interrupt context
      Use pci_[gs]et_drvdata instead of directly referenced ->driver_data in struct pci_dev.

### CODE REVIEW

**Review by tapaswenipathak — APPROVED** [review]

**Review by tapaswenipathak — COMMENTED** [review]
- `filesize_of_get_shortlogs`
- sample data files, a few, may be 1 or 2 dir
- makefile updated
- remaining transfer using drive.google.com or wetransfer
- gif of running
- those three screenshots

File: data_dir/filesizeof_get_short_logs.txt [hunk 1]
```diff
@@ -0,0 +1,496 @@
...
+39M	4.12-rc7
+676M	4.13
+40M	4.13-rc1
+40M	4.13-rc2
+40M	4.13-rc3
+40M	4.13-rc4
+40M	4.13-rc5
+40M	4.13-rc6
+40M	4.13-rc7
+13G	4.14
```
Comment (tapaswenipathak) [inline data_dir/filesizeof_get_short_logs.txt:284]: 😮

File: data_dir/filesizeof_get_short_logs.txt [hunk 2]
```diff
@@ -0,0 +1,496 @@
...
+636M	4.8
+36M	4.8-rc1
+36M	4.8-rc2
+36M	4.8-rc3
+36M	4.8-rc4
+36M	4.8-rc5
+36M	4.8-rc6
+36M	4.8-rc7
+36M	4.8-rc8
+13G	4.9
```
Comment (tapaswenipathak) [inline data_dir/filesizeof_get_short_logs.txt:404]: !

File: data_dir/filesizeof_get_short_logs.txt [hunk 3]
```diff
@@ -0,0 +1,496 @@
...
+940M	5.3
+50M	5.3-rc1
+50M	5.3-rc2
+50M	5.3-rc3
+50M	5.3-rc4
+50M	5.3-rc5
+50M	5.3-rc6
+50M	5.3-rc7
+50M	5.3-rc8
+12G	5.4
```
Comment (tapaswenipathak) [inline data_dir/filesizeof_get_short_logs.txt:447]: !!

**Comment by reddheeraj** [comment]
image
image
image

**Pull request merged**